app/src/main/java/fr/free/nrw/commons/edit/EditActivity.kt

```diff
@@ -3,11 +3,17 @@ package fr.free.nrw.commons.edit
 import android.animation.Animator
 import android.animation.Animator.AnimatorListener
 import android.animation.ValueAnimator
+import android.content.ContentResolver
+import android.content.ContentValues
 import android.content.Intent
 import android.graphics.BitmapFactory
 import android.graphics.Matrix
 import android.media.ExifInterface
```

Comment:
Looks good, few nitpicks rest good.
```diff
- import android.media.ExifInterface
+ import androidx.exifinterface.media.ExifInterface
```

Avoid using `android.media.ExifInterface`; use `androidx.exifinterface.media.ExifInterface` instead

This has been deprecated due to the security issue. Its better to replace it.[check this](https://stackoverflow.com/a/44246248)

Also IDE recommends : Avoid using `android.media.ExifInterface`; use `androidx.exifinterface.media.ExifInterface` instead